src/unitxt/eval_utils.py

```diff
@@ -1,29 +1,70 @@
 from functools import singledispatch
-from typing import List
+from typing import List, Optional
 
 import pandas as pd
 
+from .metrics import MetricPipeline
 from .operator import SequentialOperator
 from .stream import MultiStream
 
 
 @singledispatch
-def evaluate(dataset, metric_names: List[str]):
+def evaluate(
+    dataset, metric_names: List[str], compute_conf_intervals: Optional[bool] = False
+):
     """Placeholder for overloading the function, supporting both dataframe input and list input."""
     pass
 
 
 @evaluate.register
-def _(dataset: list, metric_names: List[str]):
+def _(
+    dataset: list,
+    metric_names: List[str],
+    compute_conf_intervals: Optional[bool] = False,
+):
+    global_scores = {}
     for metric_name in metric_names:
         multi_stream = MultiStream.from_iterables({"test": dataset}, copying=True)
         metrics_operator = SequentialOperator(steps=[metric_name])
+
+        if not compute_conf_intervals:
+            first_step = metrics_operator.steps[0]
+            if isinstance(first_step, MetricPipeline):
+                n_samples_before = first_step.metric.n_resamples
+                first_step.metric.n_resamples = None
+            else:
+                n_samples_before = first_step.n_resamples
+                first_step.n_resamples = None
+
         instances = list(metrics_operator(multi_stream)["test"])
         for entry, instance in zip(dataset, instances):
             entry[metric_name] = instance["score"]["instance"]["score"]
-    return dataset
+
+        if len(instances) > 0:
+            global_scores[metric_name] = instances[0]["score"].get("global", {})
+
+        if not compute_conf_intervals:
+            first_step = metrics_operator.steps[0]
```

Comment:
Please document this hack with a comment, and mention this issue https://github.com/IBM/unitxt/issues/325 in the comment. Also once this PR is merged, please add a reference from the issue to this code. With the motivation being that once the issue 325 is handled, we will remember to go back here and remove this hack.